components/formats-bsd/src/loci/formats/in/TileJPEGReader.java

```diff
@@ -58,8 +58,7 @@
 
   // -- Fields --
 
-  private JPEGTileDecoder decoder;
-  private JPEGTurboService service;
+  private transient JPEGTurboService service;
```

Comment:
If judged acceptable then we might want to think if there were any other memo breakages that we'd held that could also now go in.